libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1097,6 +1099,29 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         )
     }
 
+    /**
+     * start
+     */
+    fun startCopilot() {
+        MapboxCopilot.start()
+    }
```

Comment:
Does this have to be in `MapboxNavigation`? Why not expose `MapboxCopilot` instead, so that it can be started independently?

Reply:
Maybe it's because MapboxCopilot does not have a public interface. MapboxNavigation creates a public reference point. It also makes it so you must set up NavigationOptions before you can use copilot.

Reply:
Correct. Also, IMO having these under `MapboxNavigation` makes the feature more discoverable.

Reply:
I think having this in `MapboxNavigation` creates a false assumption that Copilot will be automatically stopped when this `MapboxNavigation` instance is destroyed, but in fact it will automatically resume when a new instance is created, because Copilot uses `MapboxNavigationObserver` under the hood.

Reply:
This is a great point 🤔 

@kmadsen and @mapbox/navigation-android any other opinions on the discussion of having Copilot within `MapboxNavigation` or not?

Reply:
I like the idea of moving Copilot API to a separate class.
If you want to keep `MapboxNavigation` one single entry point, you can at least introduce `MapboxNavigation#getCopilot` method that would return an object with all the Copilot API.
But actually `MapboxNavigation` is not a single entry point now. Look at any example: there are `MapboxRouteLineView`, `MapboxRouteLineApi`, `MapboxVoiceInstructionsPlayer` and other things. Why can't Copoilot be one of them?